.github/workflows/security.yml

```diff
@@ -28,4 +27,5 @@ jobs:
           HORUSEC_CLI_REPOSITORY_NAME: ${{ secrets.HORUSEC_CLI_REPOSITORY_NAME }}
         run: |
           curl -fsSL https://raw.githubusercontent.com/ZupIT/horusec/master/deployments/scripts/install.sh | bash -s latest
-          horusec start -p . -e true -G true
+          horusec start -p . -e true -G true --show-vulnerabilities-types="Vulnerability, Risk Accepted"
```

Comment:
What do you think about remove the "true" values? They are not necessary when using boolean flags.

Reply:
please refer to #637 since we found a bug on PR with fork origin